### PR TITLE
Remove main.pdf from tracking

### PR DESCRIPTION
**Pull request by jzzheng22**

### CODE REVIEW

**Review by fds101 — APPROVED** [review]

**Review by tomeaton17 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 ## Generated if empty string is given at "Please type another file name for output:"
 .pdf
+main.pdf
```
Comment (MattSScott) [inline .gitignore:26]: may seem super dumb - don't we just need *.pdf?

**Review by MattSScott — COMMENTED** [review]
One small thing - I might be dumb

File: .gitignore (hunk 1 above)
Comment (jzzheng22) [reply .gitignore:26]: Your team's PR has pdf files - I want to just keep it to main.pdf that isn't committed

Comment (fds101) [reply .gitignore:26]: Think it's just being specific, only the main.pdf seems to be giving problems

**Pull request merged**